Makefile

```diff
@@ -357,14 +353,15 @@ all: \
 	tests_ocaml bench_ocaml \
 	build_french_law_library_js \
 	bench_js \
-	generate_french_law_library_python type_french_law_library_python\
+	generate_french_law_library_python type_french_law_library_python \
```

Comment:
Ok I see you've moved all the logic out of the Makefile and into `dune`. That's fine by me!

Reply:
That was not strictly required for Python, but it was for js since we go through OCaml and use dune again further down the pipeline: so fo the sake of consistency…

Reply:
Yep I agree, let's try and keep things consistent.